### PR TITLE
Add ci for mappers-go

### PR DESCRIPTION
**Pull request by fisherxu**
Signed-off-by: fisherxu <xufei40@huawei.com>

Init ci test for the repo.

### CODE REVIEW

**Comment by fisherxu** [comment]
https://github.com/fisherxu/mappers-go/actions/runs/752065671

Job in my repo has run successfully, ptal

Add the lint check build, build image, need to add ut and e2e test later.

/cc @kevin-wangzefeng  @sailorvii

File: tests/e2e/mapper/bluetooth/bluetooth_test.go [hunk 1]
```diff
@@ -108,175 +97,14 @@ func ScheduleExecute(client MQTT.Client, message MQTT.Message) {
 // Checks whether mapper has written data correctly to connected device
 var _ = Describe("Application deployment test in E2E scenario", func() {
 	Context("Test write characteristic of mapper", func() {
-		BeforeEach(func() {
```
Comment (sailorvii) [inline tests/e2e/mapper/bluetooth/bluetooth_test.go:111]: Does it mean that remove those tests?

**Review by sailorvii — COMMENTED** [review]
Amazing speed for adding the CI.
I'm OK with the code-related part. However, I'm not good at the CI. Is there anyone who could help to review it?

File: tests/e2e/mapper/bluetooth/bluetooth_test.go (hunk 1 above)
Comment (fisherxu) [reply tests/e2e/mapper/bluetooth/bluetooth_test.go:111]: These tests all need refactor, I cam comment them instead of deleting.

Comment (fisherxu) [reply tests/e2e/mapper/bluetooth/bluetooth_test.go:111]: These tests all need refactor, I cam comment them instead of deleting.

**Comment by kevin-wangzefeng** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**